### ISSUE TITLE
Unable to find logical ref resource for

### ISSUE THREAD

**Issue by bboure** [ConduitVC/aws-utils#112]
@lightsofapollo 

Since #65 was merged I get this kind of error all over the place.
````
Error: Unable to find logical ref resource for : AWS::Cognito::UserPool (custom.appSync.userPoolConfig.userPoolId.Ref)
````

The reason is that the implementation only supports DynamoDb resolution for now and throws an exception on any other resource. 

This is a breaking change. Before this, `userPoolId` being a reference did not cause any problem since the emulator simply ignores/mocks authentication.

This error also occurs in any other resource referencing another, even if not even related to AppSync at all.

I suggest we change [this line](https://github.com/ConduitVC/aws-utils/blob/master/packages/appsync-emulator-serverless/cloudFormationProcessor.js#L56) to fallback to the previous behaviour and simply return `value` untouched/unresolved, instead of throwing an exception.

**Comment by pianomansam** [comment]
I'm also experiencing this issue and didn't know where it was coming from.

**Comment by lightsofapollo** [comment]
@pianomansam This should be fixed in `0.12` onward. Can you confirm this works for you?

**Comment by pianomansam** [comment]
@lightsofapollo this appears to be fixed on `0.12`
